### PR TITLE
Some minor fixes

### PR DESCRIPTION
**Pull request by abompard**
I'll comment my PR to explain why I'm suggesting this.

### CODE REVIEW

File: koji_fedoramessaging_messages/package.py [hunk 1]
```diff
@@ -60,36 +60,40 @@ class ListChangeV1(KojiFedoraMessagingMessage):
 
     @property
     def force(self) -> Optional[bool]:
-        return self.body["force"]
+        return self.body.get("force")
```
Comment (abompard) [inline koji_fedoramessaging_messages/package.py:63]: In the JSON schema, the fields are not marked in a `required` list. We have two options:
- either mark them as required, and we can access them by key in the properties
- handle their absence in the properties with a `.get()` call

I have no preference as I don't know whether Koji always gives this data.

The same applies to the rest of the file and a couple others.

File: koji_fedoramessaging_messages/repo.py [hunk 2]
```diff
@@ -43,10 +43,6 @@ class DoneV1(KojiFedoraMessagingMessage):
             },
         },
     }
-    
-    @property
-    def owner(self):
-        return None
```
Comment (abompard) [inline koji_fedoramessaging_messages/repo.py:49]: This is already the default in the base class

File: koji_fedoramessaging_messages/repo.py [hunk 3]
```diff
@@ -92,22 +88,18 @@ class InitV1(KojiFedoraMessagingMessage):
         },
     }
 
-    @property
-    def owner(self):
-        return None
```
Comment (abompard) [inline koji_fedoramessaging_messages/repo.py:97]: Same here, already the default in the base class.

File: koji_fedoramessaging_messages/rpm.py [hunk 4]
```diff
@@ -223,20 +224,27 @@ def owner(self):
...
     def sighash(self) -> str:
-        return self.body["sighash"]
+        return self.body.get("sighash")
+
+    @property
+    def packages(self) -> List[str]:
+        try:
+            return [self["build"]["package_name"]]
+        except (KeyError, TypeError):
+            return []
```
Comment (abompard) [inline koji_fedoramessaging_messages/rpm.py:250]: By reading the field description, I think it makes sense, but I'm not sure if it's always a package or if there can be modules/flatpaks/containers in there.

The `TypeError` is to handle the case where `self["build"]` is `None`.

File: koji_fedoramessaging_messages/tag.py [hunk 5]
```diff
@@ -144,44 +145,48 @@ class UntagV1(KojiFedoraMessagingMessage):
...
         return self.user
-    
+
     @property
     def usernames(self):
         return [self.agent_name, self.owner]
+
+    @property
+    def packages(self) -> List[str]:
+        return [self.name] if self.name else []
```
Comment (abompard) [inline koji_fedoramessaging_messages/tag.py:192]: Again, I think it makes sense, but I'm not sure.

File: tox.ini [hunk 6]
```diff
@@ -2,15 +2,16 @@
 envlist = lint,format,bandit,{py310,py311}-unittest
 isolated_build = true
 skip_missing_interpreters = true
+requires =
+    tox<4.0
+    poetry
+    tox-poetry
```
Comment (abompard) [inline tox.ini:8]: This tox plugin automatically builds a poetry environment for the tests.

**Comment by abompard** [comment]
I've made independent commits for my comments, hopefully that's more readable.

**Review by ryanlerch — APPROVED** [review]

**Pull request merged**